packages/create-vite/template-lit-ts/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "target": "ES2020",
-    "useDefineForClassFields": true,
+    "experimentalDecorators": true,
```

Comment:
Hi @augustjk, currently `experimentalDecorators` option is unsupported by esbuild and doesn't take any effect.

I find this pr very useful since I use Lit for my projects. 🚀